ui/de-lib/src/main/java/org/iplantc/de/client/util/OntologyUtil.java

```diff
@@ -0,0 +1,93 @@
+package org.iplantc.de.client.util;
+
+import org.iplantc.de.client.models.avu.Avu;
+import org.iplantc.de.client.models.avu.AvuAutoBeanFactory;
+import org.iplantc.de.client.models.avu.AvuList;
+import org.iplantc.de.client.models.ontologies.OntologyAutoBeanFactory;
+import org.iplantc.de.client.models.ontologies.OntologyHierarchy;
+
+import com.google.common.collect.Lists;
+import com.google.gwt.core.client.GWT;
+
+import java.util.List;
+
+/**
+ * @author aramsey
+ */
+public class OntologyUtil {
```

Comment:
I created this helper class so any `Presenter` class related to Ontologies could reuse these methods.  I also wanted to keep any messy Ontology-related String manipulation in one place, especially the Attributes, until we figure out what the long-term solution will be.